src/dmd/globals.d

```diff
@@ -141,7 +148,7 @@ struct Param
     bool release;           // build release version
     bool preservePaths;     // true means don't strip path from source file
     DiagnosticReporting warnings = DiagnosticReporting.off;  // how compiler warnings are handled
-    bool pic;               // generate position-independent-code for shared libs
+    PIC pic = PIC.fixed;    // generate fixed, pic or pie code
```

Comment:
The c++ header may be wanting this to be updated to `unsigned char` as well. Don't think it's worth mirroring the enum as well.

Reply:
I thought I'd done that already. Apparently not. Done.